ethexe/service/src/tests.rs

```diff
@@ -1483,6 +1610,8 @@ mod utils {
         pub validator_session_public_key: Option<ethexe_signer::PublicKey>,
         /// Network configuration, if provided then new node starts with network.
         pub network: Option<NodeNetworkConfig>,
+        /// RPC configuration, if provided then new node starts with RPC service.
+        pub service_rpc_config: Option<RpcConfig>,
```

Comment:
analogue with network field 2 lines above 
```suggestion
        pub rpc: Option<RpcConfig>,
```

Reply:
done